.github/workflows/global-ci-bundle.yml

```
@@ -70,7 +70,7 @@ on:
           Full JSON encoded string representing the Tackle resource to be created.
         type: string
         required: false
-        default: '{"kind":"Tackle","apiVersion":"tackle.konveyor.io/v1alpha1","metadata":{"name":"tackle"},"spec":{"image_pull_policy":"IfNotPresent","analyzer_container_memory":0,"analyzer_container_cpu":0,"feature_auth_required":false}}'
+        default: '{"kind":"Tackle","apiVersion":"tackle.konveyor.io/v1alpha1","metadata":{"name":"tackle"},"spec":{"image_pull_policy":"IfNotPresent","analyzer_container_memory":0,"analyzer_container_cpu":0,"feature_auth_required":false,"disable_maven_search": true}}'
```

```suggestion
        default: |
          kind: Tackle
          apiVersion: tackle.konveyor.io/v1alpha1
          metadata:
            name: tackle
          spec:
            image_pull_policy: IfNotPresent
            analyzer_container_memory: 0
            analyzer_container_cpu: 0
            feature_auth_required: false
            disable_maven_search: true
```
The default value can be a string with yaml content and be more readable.  It is done this way here: https://github.com/konveyor/tackle-ui-tests/blob/6da7bcd4a620b0942855ad233a535c47909f48c0/.github/workflows/run-ui-tests.yaml#L72-L83